Core/Generators/Rust/RustGenerator.cs

```diff
@@ -873,7 +1106,7 @@ private static string MakeAttrIdent(string ident)
         /// <param name="type">The field type to generate code for.</param>
         /// <param name="ot">Ownership type, e.g. <c>&'raw str</c> versus <c>String</c>.</param>
         /// <returns>The Rust type name.</returns>
-        private string TypeName(in TypeBase type, OwnershipType ot = OwnershipType.Borrowed)
+        private string TypeName(TypeBase type, OwnershipType ot)
```

Comment:
Had a few issues here as well, but I am more confident it got worked out. Conveniently I already wrote most of the code to handle "owned" types, but did not use it until now.